src/components/Footer/Footer.jsx

```diff
@@ -1,11 +1,83 @@
 import styles from './Footer.module.css';
 import clsx from 'clsx';
+import Link from 'next/link';
+import Logo from '@/icons/logo.svg';
+import Notification from '@/components/Notification';
+import { useState } from 'react';
+import subscribeTo from '@/helpers/subscribeTo';
 
 function Footer() {
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [status, setStatus] = useState('success');
+  const [sending, setSending] = useState(false);
+  const [showNotification, setShowNotification] = useState(false);
+
+  const resetNotification = () => {
+    setShowNotification(false);
+    setMessage('');
+    setStatus('');
+  };
+
+  const sendSubscription = () => {
+    setSending(true);
+    subscribeTo({ email: email })
+      .then((data) => data.json())
+      .then((data) => {
+        const { message, status: subStatus } = data;
+        setMessage(message);
+        setStatus(subStatus === 200 ? 'success' : 'fail');
+        setShowNotification(true);
+        setSending(false);
+        if (subStatus === 200) {
+          setEmail('');
+        }
+      });
+    setTimeout(() => resetNotification(), 3000);
+  };
+
+  const input = (e) => setEmail(e.target.value);
```

Comment:
`input` is not a good name for this function. It threw me off at first glance.
Something like `handleInput` sounds better.